### ISSUE TITLE
Add ability to suppress specific warning codes

### ISSUE THREAD

**Issue by pdehaan** [mozilla/addons-linter#2941]
### Describe the problem and steps to reproduce it:

Ref #2940

```sh
npx addons-linter --self-hosted ./src
```

In this case, I want to suppress all the `UNSUPPORTED_API` warnings since they aren't entirely relevant.

### What happened?

I can't find an API/flag to suppress the warnings.

### What did you expect to happen?

Hide the warnings and reduce output.

### Anything else we should know?

```sh
npx addons-linter --version # 1.14.0
```

**Comment by pdehaan** [comment]
Possible dupe of #746; _"Ability to exclude/ignore certain warnings?"_.

**Comment by Dexterp37** [comment]
I have the same requirement/problem :)
